lib/main.js

```diff
@@ -54,22 +75,24 @@ module.exports = React.createClass({
   },
 
   render: function() {
+
     var isDisabled = this.props.isDisabled;
     var tokens = this.props.selected.map(function(token) {
       return (
         Token({
           onRemove: this.handleRemove,
           value: token,
           name: token.name,
-          key: token.id})
+          key: token.id
+        })
```

Comment:
We should respect the formatting of the original library to prevent future merge conflicts